src/valuation/utils/numeric.py

```diff
@@ -155,19 +131,9 @@ def random_powerset(
         max_subsets = np.iinfo(np.int32).max
 
     def subset_probabilities(n: int) -> List[float]:
-        def sub(sizes: List[int]) -> List[float]:
-            return [math.comb(n, j) / 2**n for j in sizes]
+        return [math.comb(n, j) / 2**n for j in range(n + 1)]
 
-        job = MapReduceJob.from_fun(sub, lambda r: reduce(operator.add, r, []))
-        ret: List[List[float]] = map_reduce(job, list(range(n + 1)), num_jobs=num_jobs)
-        return ret[0]
-
-    if enable_cache:
-        _subset_probabilities = memcached(
-            client_config=client_config, cache_threshold=0.5
-        )(subset_probabilities)
-    else:
-        _subset_probabilities = subset_probabilities
+    _subset_probabilities = subset_probabilities
```

Comment:
line 136 doesn't make any sense. Also "it was unclear" doesn't seem to be reason enough to drop functionality, just because one feels it's in the way. A simple %timeit tells me that building that array for n=10e4 takes
```
37.5 s ± 91.6 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```
And this is done at _each iteration_.